modules/react/layout/lib/utils/space.ts

```diff
@@ -1,207 +1,213 @@
+   * */
+  marginX?: SystemPropValues['space'];
+  /**
+   * - sets [CSS margin-top](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-top) and [margin-bottom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-bottom)
+   * - system tokens: `space`
+   * */
+  marginY?: SystemPropValues['space'];
+  /**
+   * - sets [CSS margin-top property](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-top)
+   * - system tokens: `space`
+   * */
+  marginTop?: SystemPropValues['space'];
+  /**
+   * - sets [CSS margin-right property](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-right)
+   * - no bidirectional support
+   * - system tokens: `space`
+   * */
+  marginRight?: SystemPropValues['space'];
+  /**
+   * - sets [CSS margin-bottom property](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-bottom)
+   * - system tokens: `space`
+   * */
+  marginBottom?: SystemPropValues['space'];
+  /**
+   * - sets [CSS margin-left property](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-left)
+   * - no bidirectional support
+   * - system tokens: `space`
+   * */
+  marginLeft?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding property](https://developer.mozilla.org/en-US/docs/Web/CSS/padding)
+   * - system tokens: `space`
+   * */
+  padding?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding-left]((https://developer.mozilla.org/en-US/docs/Web/CSS/padding-left) and [padding-right properties](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-right)
+   * - system tokens: `space`
+   * */
+  paddingX?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding-top](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-top) and [padding-bottom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-bottom)
+   * - system tokens: `space`
+   * */
+  paddingY?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding-top property](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-top)
+   * - system tokens: `space`
+   * */
+  paddingTop?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding-right property](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-right)
+   * - no bidirectional support
+   * - system tokens: `space`
+   * */
+  paddingRight?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding-bottom](https://developer.mozilla.org/en-US/docs/Web/CSS/padding)
+   * - system tokens: `space`
+   * */
+  paddingBottom?: SystemPropValues['space'];
+  /**
+   * - sets [CSS padding-left property](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-left)
+   * - no bidirectional support
+   * - system tokens: `space`
+   * */
+  paddingLeft?: SystemPropValues['space'];
+};
+
+export const spaceStyleFnConfigs: StyleFnConfig[] = [
+  {
+    name: 'marginInlineStart',
+    properties: ['marginInlineStart'],
+    system: 'space',
+  },
+  {
+    name: 'marginInlineEnd',
+    properties: ['marginInlineEnd'],
```

Comment:
Again, because we dropped IE, we can use CSS Logical properties directly. We don't need to check the theme and update `marginLeft` or `marginRight` manually.